test/e2e/upgrade_test.go

```diff
@@ -95,6 +97,8 @@ func testTamperAnnotation(t *testing.T) {
 	require.NoError(t, err, "windows node upgrade failed")
 	// Test if the version annotation corresponds to the current operator version
 	testVersionAnnotation(t)
+	// Test if prometheus is reconfigured with ip address of newly configured node
+	testPrometheus(t)
```

Comment:
Don't you need a test after all nodes have been deleted to check if the enpoints are blank?